Jakub to main

ZaiZu2 opened the pull request:
Added json_set method, responsible for setting multiple keys in a json document in 1 transaction

Bugfix where get method would try to parse None value returned by Redis

ZaiZu2 committed:
Added json_set method, responsible for setting multiple keys in a json document in 1 transaction

Bugfix where get method would try to parse None value returned by Redis

DovaX reviewed (approved):
I will comment on this later

DovaX commented:
I cant merge it now

DovaX commented:
will do later

ZaiZu2 committed:
WIP on DummyRedis/Redis compatibility. Redis API abstracted into keepvariable methods

ZaiZu2 committed:
KeepVariableServers are now subclassing from Abstract class, which defines their common interface

KeepVariableDummyRedisServer now implements all methods used by the KeepVariableRedisServer, allowing it to be used in place of the latter.

ZaiZu2 committed:
Typos, Docstrings fixed

ZaiZu2 committed:
Typos corrected + additional comments

DovaX committed:
Merge branch 'master' into jakub
